From e16ee7f20d2314bf2f0536f00ae20cf0d19907c2 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 21 Mar 2024 15:17:04 +0200
Subject: [PATCH] [FSW-4100] fs1: fix fw memory addresses

There is a typo in all ARC CCM addresses, fix to correct value.

Change-Id: Ibc2c9a9a7cd062239319581cab42d7dc7b03058f
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/fs1/arc/common.h b/arch/fs1/arc/common.h
index beec0a3..6bf3e82 100644
--- a/arch/fs1/arc/common.h
+++ b/arch/fs1/arc/common.h
@@ -25,21 +25,21 @@
 #define ARC_PID_MEM_SIZE		(ARC_PID_ICCM_SIZE + ARC_PID_DCCM_SIZE)
 #define ARC_MNG_MEM_SIZE		(ARC_MNG_ICCM_SIZE + ARC_MNG_DCCM_SIZE)
 
-#define ARC_RST_ICCM_FULL_BASE		0x300007FE8800000ull
+#define ARC_RST_ICCM_FULL_BASE		0x300007FF8800000ull
 #define ARC_RST_ICCM_SIZE		SZ_64K
-#define ARC_RST_DCCM_FULL_BASE		0x300007FE8810000ull
+#define ARC_RST_DCCM_FULL_BASE		0x300007FF8810000ull
 #define ARC_RST_DCCM_SIZE		SZ_32K
-#define ARC_MNG_ICCM_FULL_BASE		0x300007FE8840000ull
+#define ARC_MNG_ICCM_FULL_BASE		0x300007FF8840000ull
 #define ARC_MNG_ICCM_SIZE		SZ_256K
-#define ARC_MNG_DCCM_FULL_BASE		0x300007FE8880000ull
+#define ARC_MNG_DCCM_FULL_BASE		0x300007FF8880000ull
 #define ARC_MNG_DCCM_SIZE		SZ_128K
-#define ARC_PID_ICCM_FULL_BASE		0x300007FE88C0000ull
+#define ARC_PID_ICCM_FULL_BASE		0x300007FF88C0000ull
 #define ARC_PID_ICCM_SIZE		SZ_128K
-#define ARC_PID_DCCM_FULL_BASE		0x300007FE88E0000ull
+#define ARC_PID_DCCM_FULL_BASE		0x300007FF88E0000ull
 #define ARC_PID_DCCM_SIZE		SZ_64K
-#define ARC_MISC_ICCM_FULL_BASE		0x300007FE8900000ull
+#define ARC_MISC_ICCM_FULL_BASE		0x300007FF8900000ull
 #define ARC_MISC_ICCM_SIZE		SZ_256K
-#define ARC_MISC_DCCM_FULL_BASE		0x300007FE8940000ull
+#define ARC_MISC_DCCM_FULL_BASE		0x300007FF8940000ull
 #define ARC_MISC_DCCM_SIZE		SZ_64K
 
 /* CFG is at Arc region F, specs CFG base address is 0x70000000. 7+8 = 0xF */
